scripts/configure-httpd-ipa.sh

```diff
@@ -16,7 +16,7 @@ else
     IRONIC_BASE_URL="http://${IRONIC_URL_HOST}"
 fi
 
-INSPECTOR_EXTRA_ARGS=" ipa-inspection-callback-url=${IRONIC_BASE_URL}:${IRONIC_INSPECTOR_ACCESS_PORT}/v1/continue"
+export INSPECTOR_EXTRA_ARGS=" ipa-inspection-callback-url=${IRONIC_BASE_URL}:${IRONIC_INSPECTOR_ACCESS_PORT}/v1/continue"
```

Comment:
let's leave this untouched and instead export the variable after the if-then block, in L23 for example

Reply:
@elfosardo 
I don't understand why you have to move to L23.

Reply:
because we're modifyinf the variable in L21

Reply:
@elfosardo 
ok, i changed it.

Reply:
https://github.com/metal3-io/ironic-image/pull/388/files#diff-ae3e5b5ce83ded18509d59dd21be8cf99921107ef4ab4d571dff31dd76c581caR23